book/src/hello_world_ethereum.md

```diff
@@ -0,0 +1,41 @@
+# Hello World on Ethereum
+
+This example is a variation of the previous [Hello World](./hello_world.md),
+targeting verification on Ethereum. In this example we will cover how to
+generate proofs directly using a Keccak transcript instead of the Poseidon
+transcript of the previous example, which will enable us to verify proofs
+onchain. There are almost no differences from the CLI perspective.
+
+```
+{{#include ../../test_data/asm/book/hello_world.asm}}
+```
+
+```console
+powdr pil test_data/asm/book/hello_world.asm --field bn254 --inputs 0 --prove-with halo2 --backend-options "snark_single" -f --params params.bin
+```
+
+The extra parameter `backend-options` tells powdr to produce a single
+SNARK that uses Keccak. The option `-f` forces overwriting the compiled files
```

Comment:
What do you mean by "that uses Keccak"?

Reply:
Keccak transcript as opposed to Poseidon transcript